packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -278,6 +307,15 @@ export const Loading: StoryObj<AutocompleteType> = {
     isLoading: true,
     options: [],
   },
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+    const comboBoxElement = canvas.getByRole("combobox") as HTMLInputElement;
+    await step("Click for loading to be visible", async () => {
+      await userEvent.click(comboBoxElement);
+      const loadingElement = screen.getByText("Loading…");
+      await expect(loadingElement).toBeVisible();
```

Comment:
We shouldn't need `await` here:
```suggestion
     expect(loadingElement).toBeVisible();
```

From looking online, only `.resolve` as in `expect().resolve().toBe()` will return a promise.